README.md

```diff
@@ -19,6 +19,8 @@ Before you can use the Open Commerce CLI, ensure you have all the base requireme
 - [Docker](https://www.docker.com/get-started/)
 - [Docker Compose](https://docs.docker.com/compose/)
 
+In addition, you need to have your system setup for [SSH authentication with GitHub](https://docs.github.com/en/authentication/connecting-to-github-with-ssh)
```

Comment:
Why do we actually need configure github ssh configured? Also, isn't that implicitly required if you are using GitHub anyway?

Reply:
Because we clone repos in the background to pull the files. Lots of people ask about this so I want to make it explicit

Reply:
Aren't we cloning everything required over http rather than ssh? Usually ssh is used when the user intend to push to the the remote. In the case of the CLI, the user only pull so http is sufficient.

Reply:
Aren't we cloning everything required over HTTP rather than ssh? Usually, ssh is used when the user intends to push to the remote. In the case of the CLI, the user only pulls so HTTP is sufficient.

Reply:
No.